diffrax/_brownian/tree.py

```diff
@@ -222,31 +223,31 @@ def evaluate(
         t1: Optional[RealScalarLike] = None,
         left: bool = True,
         use_levy: bool = False,
-    ) -> Union[PyTree[Array], LevyVal]:
-        def _is_levy_val(obj):
-            return isinstance(obj, LevyVal)
+    ) -> Union[PyTree[Array], Union[TimeLevyArea, SpaceTimeLevyArea]]:
+        def _is_levy_area(obj):
+            return isinstance(obj, Union[TimeLevyArea, SpaceTimeLevyArea])
```

Comment:
Note that `isinstance(..., Union[...])` are buggy across different Python versions. (If you're curious, I go on [a bit of a rant about this here](https://github.com/patrick-kidger/jaxtyping/issues/73#issuecomment-1493431459).) Better to use tuples instead:
```python
isinstance(obj, (TimeLevyArea, SpaceTimeLevyArea))
```

Reply:
I was not aware of this. Your post was a good read thank you!